topologies/single.go

```diff
@@ -36,6 +36,16 @@ func (t *SingleTopology) IsRouteReflector(_ string, labels map[string]string) bo
 	return ok && label == t.NodeLabelValue
 }
 
+func (t *SingleTopology) IsMultiZone(nodes map[*corev1.Node]bool) bool {
```

Comment:
Current implementation checks `ROUTE_REFLECTOR_ZONE_LABEL` if has value the cluster is multi zone. i don't think we need to interate over all the nodes.